src/calculator.js

```diff
@@ -75,33 +75,31 @@ for (var i = 0; i < operationButtons.length; i++) {
     }
 }
 
-var memo = new Memory();
-var memoryPlus = document.getElementById('memoryPlus');
+let memo = new Memory();
+let memoryPlus = document.getElementById('memoryPlus');
 memoryPlus.onclick = function() {
-    var currNumber = screen.getNumber();
-    memo.Plus(Number(currNumber));
-    screen.resetOnNextInput();
+    let currNumber = screen.getNumber();
+    memo.plus(Number(currNumber));
```

Comment:
Here is the problem. You return a string from `getNumber()` method, which is bad. Due to this fact you need to parse it using `Number(currNumber)` later, which should be unnecessary.

If a method is called `getNumber`, **it needs to return a number**.